cognite_toolkit/_cdf_tk/commands/_changes.py

```diff
@@ -64,7 +64,7 @@ class SystemYAMLMoved(AutomaticChange):
     has_file_changes = True
 
     def do(self) -> set[Path]:
-        system_yaml = self._organization_dir / COGNITE_MODULES / "_system.yaml"
+        system_yaml = self._organization_dir / BUILTIN_MODULES / "_system.yaml"
```

Comment:
This an automatic upgrade of an earlier module, and should not be changed, as it will break `cdf mdules upgrade`
```suggestion
        system_yaml = self._organization_dir /  "cognite_modules" / "_system.yaml"
```